requirements_all.txt

```diff
@@ -479,6 +479,9 @@ eliqonline==1.2.2
 # homeassistant.components.elkm1
 elkm1-lib==0.7.15
 
+# homeassistant.components.elmo_alarm
+elmo==0.1.2
```

Comment:
Elmo client maintainer here! @markin I've integrated your latest changes in the upstream and you can use the [official package](https://pypi.org/project/econnect-python/) `econnect-python` instead of using the fork.

```suggestion
# homeassistant.components.elmo_alarm
econnect-python==0.3.1
```